3p/remote.html

```diff
@@ -4,9 +4,22 @@
 <meta name="robots" content="noindex">
 <script>
 (function() {
-var v = location.search.substr(1);
 if (!(/^\d+(-canary)?$/.test(v))) return;
-var u = 'https://3p.ampproject.net/'+encodeURIComponent(v)+'/f.js';
+var c = window.AMP_CONFIG = window.AMP_CONFIG || {};
```

Comment:
Yes.  That is why want to go get the new one out and then we will go encourage publishers to update.